client-js/TUTORIAL.md

```diff
@@ -0,0 +1,202 @@
+      meta: {
+        color: "Blue",
+        size: "Medium",
+        material: "Aluminum",
+        condition: "Used",
+      },
+    },
+    "1000000000",
+    FAUCET_KEYS.publicKey,
+    [FAUCET_KEYS]
+  );
+
+  const mintDeployHash = await mintDeploy.send(NODE_URL!);
+
+```
+The arguments adhere to those provided in the original installation.
+
+## Transferring a Token
+
+After minting one or more tokens, you can then use the following code to transfer the tokens between accounts:
+
+```js
+
+  const transferDeploy = await contractClient.transfer(
+    {
+      tokenId: "0",
+      source: FAUCET_KEYS.publicKey,
+      target: USER1_KEYS.publicKey,
+    },
+    "13000000000",
+    FAUCET_KEYS.publicKey,
+    [FAUCET_KEYS]
+  );
+
+  const transferDeployHash = await transferDeploy.send(NODE_URL!);
+
+```
+
+Transferring accepts the following arguments:
+
+* `tokenId` - The sequential ID assigned to a token in mint order.
+
+* `source` - The account sending the token in question.
+
+* `target` - The account receiving the transferred token.
+
+## Burning a Token
+
+The following code shows how to burn a minted NFT that you hold and have access rights to, requiring only the `tokenId` argument:
+
+```js
+
+  const burnDeploy = await contractClient.burn(
+    { tokenId: "0" },
+    "13000000000",
+    USER1_KEYS.publicKey,
+    [USER1_KEYS]
+  );
+
+  const burnDeployHash = await burnDeploy.send(NODE_URL!);
+
+```
+
+## Testing
+
+### Running an Install Test
+
+This repository includes a test script for installing a CEP-78 contract instance.
+
+You will need to define the following variables in the `.env` file:
+
+* `NODE_URL` - The address of a node. If you are testing using [NCTL](https://docs.casperlabs.io/dapp-dev-guide/building-dapps/setup-nctl/), this will be `http://localhost:11101/rpc`.
+
+* `NETWORK_NAME` - The name of the Casper network you are testing on, `casper-net-1` when testing using a local network with NCTL.
+
+* `MASTER_KEY_PAIR_PATH` - The path to the key pair of the minting account.
+
+* `USER1_KEY_PAIR_PATH` - The path to an additional account's key pair for use in testing transfer features.
+
+You may also need to install associated dependencies using:
+
+```js
+npm i
+```
+
+This test can be run using the following command:
+
+```js
+npm run test:install
+```
+
+The test will then return the installation's `deployHash`, and inform you when the installation is successful.
+
+The test will then provide the installing account's information, which will include the CEP-78 NFT contract's hash and package hash.
+
+
+### Running a Usage Test
+
+A usage test uses the same variables as the Install test above, but tests the basic functionality of the contract after installation.
+
+The usage test can be run using the following command:
+
+```js
+npm run test:usage
+```
+
+This test will acquire the contract's hash and package hash, prior to sending three separate deploys to perform several function tests as follows:
```

Comment:
In this section I would replace `test` for `example` everywhere. Also we can point users that they are running code that is in `client-js/examples` directory so they can treat it as reference :)